proxy/coco/src/keystore.rs

```diff
@@ -108,10 +108,10 @@ impl Keystorage<Memory> {
 }
 
 /// Synonym for an error when interacting with a file-backed store for [`librad::keys`].
-type FileError = file::Error<SecretBoxError<Infallible>, keys::IntoSecretKeyError>;
```

Comment:
Note that I made these `pub` because `cargo doc` started complaining. I see no harm in them being public